src/test/java/ru/training/at/hw1/Tags.java

```diff
@@ -0,0 +1,6 @@
+package ru.training.at.hw1;
+
+public interface Tags {
+    String ADD_SUB = "addSub";
+    String MULT_DIV = "multDiv";
```

Comment:
I see the names of the constants, but don't see appropriate keywords for a declaration.

Reply:
Fixed.